frontend/src/app/Settings/Users/ManageUsers.test.tsx

```diff
@@ -494,7 +494,6 @@ describe("ManageUsers", () => {
       await userEvent.click(resetButton);
       const sureButton = await screen.findByRole("button", {
         name: "Reset multi-factor authentication",
-        exact: false,
```

Comment:
⚠️ `@testing-library/react` updated `@testing-library/dom` to v9 which introduced a breaking change here: https://github.com/testing-library/dom-testing-library/releases/tag/v9.0.0

tldr; any `byRole` queries dropped support for "exact, trim, collapseWhitespace, and normalizer options."